### ISSUE TITLE
Quickfile button behaves different if nav not the current window

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ijt/emacs-nav#26]
```
What steps will reproduce the problem?
1. Make current any window besides nav
2. Click quickfile button with mouse
3. Will open file in a new frame 

What is the expected output? 

The file should up in a new buffer under current window.


nav-quickdir-jump-button-action function needs to test if
nav is the current window. If not make it the current. 
```

Original issue reported on code.google.com by `matthew....@gmail.com` on 29 Jul 2009 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
This seems to happen in GNU Emacs 23 but not in GNU Emacs 22.
```

Original comment by `matthew....@gmail.com` on 29 Jul 2009 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
Fixed by rev 49bfb8f5fc.
```

Original comment by `issac.tr...@gmail.com` on 30 Jul 2009 at 7:29
- Changed state: **Fixed**
